src/v/cloud_storage/partition_manifest.h

```diff
@@ -547,6 +572,8 @@ class partition_manifest : public base_manifest {
     kafka::offset _start_kafka_offset;
     // Total size of the archive (excluding this manifest)
     uint64_t _archive_size_bytes{0};
+    /// Map of spillover manifests that were uploaded to S3
+    spillover_manifest_map _spillover_manifests;
```

Comment:
Don't we need to persist this list in the snapshot? Or perhaps we can bring back the list-objects code and use it in that scenario.

Reply:
fixed